runtimelib/src/jupyter/client.rs

```diff
@@ -374,22 +392,13 @@ pub struct JupyterClient {
 impl JupyterClient {
     /// Close all connections to the kernel
     pub async fn detach(self) -> Result<()> {
-        let timeout_duration = Duration::from_millis(60);
-
-        let close_sockets = async {
-            let _ = tokio::join!(
-                self.shell.socket.close(),
-                self.iopub.socket.close(),
-                self.stdin.socket.close(),
-                self.control.socket.close(),
-                self.heartbeat.socket.close(),
-            );
-        };
-
-        match timeout(timeout_duration, close_sockets).await {
-            Ok(_) => Ok(()),
-            Err(_) => Err(anyhow!("Timeout reached while closing sockets.")),
-        }
+        // todo!(): Bring back timeout duration or move this to drop
```

Comment:
Should this be elevated to an issue? I don't fully understand the necessity of the close on detach in terms of the protocol.

Reply:
It's back in now!